sketch.js

```diff
@@ -176,4 +202,56 @@ function mousePressed() {
 // Fade it out when we release
 function mouseReleased() {
   osc.stop(release + 1);
+}
+
+//When we press down a key (Ignores action keys)
+function keyTyped() {
+  for(var keyboardKey = 0; keyboardKey < keyboardKeys.length; keyboardKey++) {
```

Comment:
Any reason you're not using `for(var keyboardKey in keyboardKeys)`?

Also since the var isn't actually the key, but rather the index into the array, so about about `for(var keyIndex in keyboardKeys)`